modules/example/src/smithy4s/example/StructureWithRefinedMember.scala

```diff
@@ -0,0 +1,23 @@
+package smithy4s.example
+
+import smithy4s.Hints
+import smithy4s.Schema
+import smithy4s.ShapeId
+import smithy4s.ShapeTag
+import smithy4s.example.refined.Age
+import smithy4s.example.refined.Age.provider._
+import smithy4s.schema.Schema.int
+import smithy4s.schema.Schema.struct
+
+case class StructureWithRefinedMember(otherAge: Option[Age] = None)
```

Comment:
added another structure instead of messing with the other one, but I opened an issue to track what I've saw in [issue #791](https://github.com/disneystreaming/smithy4s/issues/791)